docs/readmes/howtos/ue_ip_address_management.md

```diff
@@ -20,11 +20,11 @@ update these values if you don't have access to the API. Navigate to the
 corresponding component on the NMS (e.g. gateway, network, subscriber) and
 use the JSON editor to edit its configuration.
 
-## Enable Bridged Mode
+## Enable Non-NAT Mode
 
-To configure an AGW in bridged mode, you need to disable NAT on the AGW.
+To configure an AGW in Non-NAT mode, you need to disable NAT on the AGW.
 Note that this is a gateway-level configuration, so you will need to change
-this for every AGW in the network that you want to run in bridged mode.
+this for every AGW in the network that you want to run in Non-NAT mode.
```

Comment:
Can we explain what the network wide flag for "NAT" vs "DHCP" does? I think there was some confusion there when we were debugging this with @sourabh-nanoti also.

Reply:
That is explained in the 'UE IP Address Allocation' section of this doc.
I know its confusing, in next release we can add better option for IP allocation in APN API restructuring work.